android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -41,6 +45,26 @@ public Map<String, Object> getExportedCustomDirectEventTypeConstants()
                 .build();
     }
 
+    @Nullable
+    @Override
+    public Map<String, Integer> getCommandsMap()
+    {
+        return MapBuilder.of(
+                "loadAd", COMMAND_LOAD_AD
+        );
+    }
+
+    @Override
+    public void receiveCommand(@NonNull final AppLovinMAXAdView adView, final int commandId, @Nullable final ReadableArray args)
```

Comment:
I just noticed you name the `AppLovinMAXAdView` param as `view` everywhere else in this file, so to be consistent let's keep that.